rust-crates/symblib-capi/src/gosym.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{FfiResult, StatusCode, SymblibPointResolver};
+use std::ffi::{c_char, CStr};
+use std::path::Path;
+use symblib::symbconv::PointResolver;
+use symblib::{gosym::GoRuntimeInfo, objfile};
+
+pub struct SymblibGoRuntime {
+    #[allow(dead_code)]
+    runtime: GoRuntimeInfo<'static>,
+}
```

Comment:
Not used, as far as I can tell? We may have to bring it back, though -- see other comment.